Makefile

```diff
@@ -69,15 +69,53 @@ vhs.sh:
 # Web #
 #######
 
-## Web - Start web server (PORT)
-web: PORT = 9400
-web:
-	echo Start web server...
+## Web - Install
+web.install:
+	go install
+	cd web/app && make install
+.PHONY: install
+
+web: web.serve
+
+## Web - Start api web server + React app (PORT)
+web.serve: PORT ?= 9400
+web.serve:
+	# https://www.npmjs.com/package/concurrently
+	npx concurrently "PORT=$(PORT) make web.serve.api+local" "API_PORT=$(PORT) make web.serve.front" \
+		--names="API,Front" \
+		--prefix=name \
+		--prefix-colors="auto" \
+ 		--kill-others \
+ 		--kill-others-on-fail \
+ 		--colors
+.PHONY: web.serve
+
+## Web - Start api web server only (PORT)
+web.serve.api+local: PORT ?= 9400
```

Comment:
```suggestion
web.serve.api+local: PORT = 9400
```

Reply:
Wy do you want me to change this? 🤔 
It allows to override the value with:

```shell
PORT=9300 make web.serve.api+local
```

which does not work without `?=`

Reply:
Well, traditionally, in makefiles, parameters are passed using args, not envs.
`PORT=9300 make web.serve.api+local` -> `make web.serve.api+local PORT=9300`.

In the second - let's say more idiomatic - case, it should work without `?`

Reply:
Alright 👌🏻